sql/sys_vars.cc

```diff
@@ -633,6 +633,22 @@ static Sys_var_charptr Sys_character_sets_dir(
        READ_ONLY GLOBAL_VAR(charsets_dir), CMD_LINE(REQUIRED_ARG),
        IN_FS_CHARSET, DEFAULT(0));
 
+static bool check_unsupported_engine(sys_var *self, THD *thd, set_var *var)
```

Comment:
please, rename to something more specific. For example, `check_engine_supports_temporary`